backend/src/database/models/organization.ts

```diff
@@ -88,6 +88,46 @@ export default (sequelize) => {
         defaultValue: false,
         allowNull: false,
       },
+      founded: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+      },
+      industry: {
+        type: DataTypes.TEXT,
+        allowNull: true,
+      },
+      size: {
+        type: DataTypes.TEXT,
+        allowNull: true,
+      },
+      naics: {
+        type: DataTypes.ARRAY(DataTypes.JSONB),
+        allowNull: true,
+      },
+      profiles: {
```

Comment:
How does this differ from github, linkedin, crunchbase etc?

Reply:
The profile holds all the social networks a company has. It is designed to hold multiple links to the same network. It is part of the requirement in the task document.